src/main/groovy/org/mockito/release/internal/gradle/DefaultReleaseNotesPlugin.java

```diff
@@ -27,13 +27,14 @@
     public static final String TEMP_SERIALIZED_NOTES_FILE = "/notableReleaseNotes.ser";
 
     public void apply(final Project project) {
-        final DefaultReleaseNotesExtension notes = new DefaultReleaseNotesExtension(project.getProjectDir(), EXTENSION_NAME);
+        final DefaultReleaseNotesExtension notes = new DefaultReleaseNotesExtension(project.getProjectDir(), project.getBuildDir(), EXTENSION_NAME);
 
         //TODO those should be task classes with decent API
         project.getTasks().create("updateReleaseNotes", new Action<Task>() {
             public void execute(Task task) {
                 task.setGroup(TASK_GROUP);
                 task.setDescription("Updates release notes file.");
+                task.dependsOn("fetchContributorsFromGitHub");
```

Comment:
Since you assume that "fetchContributors..." task is present here, this plugin (DefaultReleaseNotesPlugin) should apply Contributors plugin. Otherwise, the release notes plugin is not useful on its own - it would break if somebody applies it without applying contributors plugin.

Hope it makes sense, I updated the code in my refactoring push, to give some guidance on this!